www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -53,15 +48,13 @@ include all iPad® models, iPhone® 3GS and above, and iPod® Touch 3rd
 Generation or later. To install apps onto a device, you must also be a
```

Comment:
You no longer need a paid account to install apps on real devices, you need xcode 7